transformerlab/plugins/fastchat_server/model_worker.py

```diff
@@ -566,6 +566,67 @@ def compute_attention_entropy(attentions):
     return final_np_array
 
 
+@app.get("/architecture_available")
```

Comment:
Shouldn't this endpoint name be more specific than "architecture available" for example like supports_visualization or supports_architecture_visualization